tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -288,6 +299,33 @@ void test_extract() {
     assert(ranges::equal(cont, elements));
 }
 
+// TRANSITION, too simple
+void test_erase_1() {
+    flat_set<int> fs{1};
+    fs.erase(1);

```

Comment:
This was rejected by the compiler, as `_Erase(auto&&)` got `const key&` which was not same as `key`.